File: 2137-final-value-of-variable-after-performing-operations/final-value-of-variable-after-performing-operations.ts
function finalValueAfterOperations(operations: string[]): number {
    let result: number = 0;
    for (const operation of operations) {
        switch (operation[1]) {
            case '+':
                result += 1;
                break;
            default:
                result -= 1;
        }
    }
    return result;
};

File: 2137-final-value-of-variable-after-performing-operations/final-value-of-variable-after-performing-operations.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int res = 0;
        for (String operation : operations) {
            switch (operation.charAt(1)) {
                case '+':
                    res += 1;
                    break;
                default:
                    res -= 1;
            }
        }
        return res;
    }
}